example/src/Label/LabelPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { Label, Screen, useTheme } from 'smartway-react-native-ui';
+
+export const LabelPage = () => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        container: {
+            alignItems: 'center',
+        },
+        label: {
+            margin: theme.sw.spacing.xs,
+        },
+    });
+
+    return (
+        <Screen style={styles.container}>
+            <Label
+                style={styles.label}
+                bgColor={theme.sw.colors.secondary[200]}
+                textColor={theme.sw.colors.secondary[600]}
+                text="Label"
+            />
```

Comment:
I have a question I dont find it in the design system but maybe we will have predefine label style like:
```
 <Label mode='info'>
 <Label mode='error'>
```

or it will be custom style everytime ?

Reply:
We do have 3 variants:
- outlined
- filled
- soft

And we have the following colors available:
- Error
- Warning
- Success
- Info
- Secondary
- Primary
- Default